src/uwtools/drivers/driver.py

```diff
@@ -463,7 +482,7 @@ def _runscript_path(self) -> Path:
         """
         The path to the runscript.
         """
-        return self.rundir / f"runscript.{self.driver_name}"
+        return self.rundir / f"runscript.{self.driver_name()}"
```

Comment:
This is a frustrating breaking change. I see that the combined use of `@classmethod` and `@property` is being removed in Python 3.13.

Reply:
Frustrating breaking change by us in `uwtools`, or by them in Python? Both maybe. I'll need to make a tiny update to the Coastal driver due to this, but I don't think we have much exposure on this outside our own drivers yet.

Reply:
Both. But more so in Python.